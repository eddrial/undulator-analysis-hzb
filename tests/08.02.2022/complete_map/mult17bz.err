 error report for the data set mult17bz.dvm        
--------------------------------------------
      
         127   data points thrown away to get a
         monotonically increasing data set
 the truncated data set has only   1591 data points 
        out of totally   2558 data points
 the derivative at  0.42895E+03 and  0.45261E+03 is zero 
 the derivative at  0.50754E+03 and  0.53881E+03 is zero 
 the derivative at  0.59247E+03 and  0.62332E+03 is zero 
 the derivative at  0.62332E+03 and  0.64825E+03 is zero 
 the derivative at  0.64825E+03 and  0.67910E+03 is zero 
 the derivative at  0.67910E+03 and  0.70318E+03 is zero 
 the derivative at  0.95544E+03 and  0.98628E+03 is zero 
 the derivative at  0.98628E+03 and  0.10395E+04 is zero 
 the derivative at  0.10395E+04 and  0.10632E+04 is zero 
 the derivative at  0.10632E+04 and  0.10945E+04 is zero 
 the derivative at  0.11202E+04 and  0.11519E+04 is zero 
 the derivative at  0.12381E+04 and  0.12559E+04 is zero 
 the derivative at  0.12559E+04 and  0.12892E+04 is zero 
 the derivative at  0.13171E+04 and  0.13484E+04 is zero 
 the derivative at  0.14321E+04 and  0.14633E+04 is zero 
 the derivative at  0.16835E+04 and  0.17147E+04 is zero 
 the derivative at  0.18225E+04 and  0.18470E+04 is zero 
 the derivative at  0.18470E+04 and  0.18821E+04 is zero 
 the derivative at  0.20464E+04 and  0.20798E+04 is zero 
 the rms value of the residuals is  0.25079E-02
 the maximum residual is  0.11371E-01
 the rms value of the lambda/2-filtered data is  0.13584E-02
 the maximum value of the lambda/2-filtered data is  0.63213E-02
