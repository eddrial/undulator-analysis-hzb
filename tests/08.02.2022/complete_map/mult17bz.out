mult17bz.dvm
            

 names of parameters as follows : 
b(1) == amplitude, 1st harm.
b(2) == amplitude, 3rd harm.
b(3) == amplitude, 5rd harm.
....                        
b(n) == amplitude, n^th harm.
b(n+1) == lambda, mm.
b(n+2) == x0, 1st harm, mm.
b(n+3) == x0, 3rd harm, mm.
b(n+4) == x0, 5th harm, mm.
....
b(nparm) == x0, n^th harm, mm.

 parameters:
   1 -9.0100942E-03
   2  1.1526646E-04
   3  1.7948293E-04
   4  9.2737437E-06
   5  8.7371404E-07
   6  5.6003716E+01
   7  1.0408630E+03
   8  1.0410413E+03
   9  1.0440697E+03
  10 -9.0275555E+00
  11 -5.5675764E+00
 number of iterations:   10
     sumsq:  1.00064E-02
       rms:  2.50786E-03  dimension of magnetic field
       rms:  2.78334E+01  percent of b_eff
   rms_ext:  2.22488E+01  percent of b_eff
 B_eff =   9.0102E-03
 B_max =  -8.9339E-03
 K_eff =    0.047
 lamda_1 (nm) =      2.54310

        eigenvectors in parameter space

 error                         direction
   101.17,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00
    12.25,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00
     2.30,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00
     0.89,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00
     0.10,  0.00  0.00  0.00  0.00  0.00 -0.05  1.00  0.00  0.00  0.00  0.00
     0.01,  0.00  0.00  0.00  0.00  0.00  1.00  0.05  0.00  0.00  0.00  0.00
     0.00, -0.51  0.34  0.41  0.60  0.31  0.00  0.00  0.00  0.00  0.00  0.00
     0.00,  0.40  0.59  0.56 -0.18 -0.38  0.00  0.00  0.00  0.00  0.00  0.00
     0.00,  0.64  0.30 -0.41  0.49  0.30  0.00  0.00  0.00  0.00  0.00  0.00
     0.00,  0.17 -0.01  0.29 -0.48  0.81  0.00  0.00  0.00  0.00  0.00  0.00
     0.00, -0.38  0.67 -0.52 -0.38  0.05  0.00  0.00  0.00  0.00  0.00  0.00
   
  calculated kicks for trajectory compensation: 
 kick_1:  -0.05750 Tmm
 kick_2:   0.05750 Tmm
      
 the rms value of the second derivative is  0.00000E+00
 the maximum value of the second derivative is  0.00000E+00
      
 the maximum transverse movement of the electron in mu_meter
                  (peak to peak) is  0.35722E+01
      
 the truncated data set has only   1591 data points 
      
 the rms value of the residuals is  0.25079E-02
 the maximum residual is  0.11371E-01
      
 the rms value of the lambda/2-filtered data is  0.13584E-02
 the maximum value of the lambda/2-filtered data is  0.63213E-02
      
 the rms phase error (all data) is    0.509 deg.
 the maximum phase error (all data) is    2.937 deg.
      
 the rms phase error (extrema) is    2.626 deg.
 the maximum phase error (extrema) is    5.311 deg.
      
 the Walker-rms phase error (all data) is    0.511 deg.
 the Walker-maximum phase error (all data) is    3.031 deg.
      
 the Walker-rms phase error (extrema) is    0.473 deg.
 the Walker-maximum phase error (extrema) is    1.099 deg.
  
  phase: straight line subtracted
 -----------------------------------------
 total phase error            0.5085002    
 local phase error            0.2951850    
 slowly varying phase error   0.4114624    
 oscillatory phase error      4.6226580E-02
 -----------------------------------------
